src/oc_erchef/apps/chef_index/README.md

```diff
@@ -6,7 +6,7 @@ Chef is a system integration framework written in erlang and ruby and designed t
 
 The Chef Wiki is the definitive source of user documentation.
 
-    http://wiki.chef.io/display/chef/Home
```

Comment:
Oh good point.  Yeah f-that.  We should have automate notifications in a good enough place to ask anyone upgrading to ES5 to use that instead of reporting. #famouslastwords.

Heads up to @charlesjohnson.